src/components/BarChartCard/BarChartCard.story.jsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { text, select, object, boolean } from '@storybook/addon-knobs';
+
+import {
+  CARD_SIZES,
+  BAR_CHART_TYPES,
+  BAR_CHART_ORIENTATION,
+} from '../../constants/LayoutConstants';
+import { getCardMinSize } from '../../utils/componentUtilityFunctions';
+import { barChartData } from '../../utils/sample';
+
+import BarChartCard from './BarChartCard';
+
+storiesOf('Watson IoT Experimental|BarChartCard', module)
+  .add('Simple bar', () => {
+    const size = select('size', Object.keys(CARD_SIZES), CARD_SIZES.WIDE);
```

Comment:
The `XSMALL` size is essentially useless as its too small to see any of the data. Should a warning be added here to not use it? Or maybe not show it as an option at all?

Reply:
I'd vote for a prop-types validation on the XSMALL size...  Also needs to be updated to match the newer card sizes that @JoelArmendariz just delivered to dev

Reply:
done